### ISSUE TITLE
Taskbar Widget NetworkManager Applet: Stays open

### ISSUE THREAD

**Issue by RefinedSoftwareLLC** [QubesOS/qubes-issues#3998]
### Qubes OS version:

R4.0

### Affected component(s): 

dom0's Taskbar Widget "NetworkManager Applet"

---

### Steps to reproduce the behavior:

Click open dom0's Taskbar Widget "NetworkManager Applet", now click on any other widget or VM.
"NetworkManager Applet" doesn't automatically close.

### Expected behavior:

"NetworkManager Applet" close like the other widgets when another widget or VM is clicked on.

**Comment by marmarek** [comment]
This is result of GUI isolation - "NetworkManager Applet", running in sys-net doesn't see clicks outside of its window, so it doesn't close when you click anything which doesn't belong to sys-net.
But regardless of this, it is a valid UX issue.

**Comment by RefinedSoftwareLLC** [comment]
I solved this issue in a GUI manager I designed once.
When the GUI manager (in your case dom0) detects the window losing focus, normally by any mouse clicks or keyboard key presses outside the window, it runs a clear-all-input function for that window. This function can't be called again until the window has regained focus. This function would move the mouse  offscreen to a location no window is under, then release any held mouse or keyboard buttons.

You many want to use your Salt Devops to tag this window (sys-net's NetworkManger Applet windows). This tag would mean that when the "clear-all-input" function is triggered (tagged window loses focus), after moving the mouse offscreen to a location no window is under, dom0 then sends an emulated mouse click & release (or an "Esc" keypress & keyrelease).

Once again, for security purposes, these emulated inputs can only happen once per the VM window losing focus, so they can't eavesdrop on inputs from other VMs, beyond knowing when they lost focus.
